subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/internal/AbstractCodeQualityPlugin.java

```diff
@@ -121,9 +123,10 @@ private Map<String, String> excludeProperties(String group, String module) {
 
     protected abstract CodeQualityExtension createExtension();
 
+    @SuppressWarnings("rawtypes")
```

Comment:
Did you fix the raw types in the method? Which one is still left?

Reply:
The raw type warning comes from `Plugin`. It can't be changed to `Plugin<?>` because it's in a public interface method `withBasePlugin`.